src/xFactor/infrastructure/dao/JudgeDao.java

```diff
@@ -0,0 +1,23 @@
+package xFactor.infrastructure.dao;
+
+import org.hibernate.Session;
+
+import xFactor.infrastructure.model.Judge;
+
+public class JudgeDao extends BaseDao {

```

Comment:
you can try make BaseDao generic class and then save, update, find and etc will be extracted there